examples/complete/main.tf

```diff
@@ -55,6 +57,25 @@ module "log_bucket" {
   attach_elb_log_delivery_policy = true
 }
 
+module "cloudfront_log_bucket" {
+  source = "../../"
+
+  bucket = "cloudfront-logs-${random_pet.this.id}"
+  acl    = null # conflicts with default of `acl = "private"` so set to null to use grants
```

Comment:
this does work; since `acl` and `grant` conflict (and you will get a message stating as much unless you add this). the alternative would be to change the default for `acl` to be `""` or `null`, but that would be a breaking change so this seemed like a better compromise. maybe there is a place to better document this? open to suggestions